tasks/Logger.cpp

```diff
@@ -100,6 +102,12 @@ void Logger::updateHook()
             it->typelib_marshaller->marshal(it->logger->getStream(), it->marshalling_handle);
         }
     }
+
+    base::Time now = base::Time::now();
```

Comment:
Does it make sense to use the "stamp" variable instead?

Reply:
Given that in between stamp and here we're actually writing to disk, I thought better to take the time again. Ultimately, it's not that important, so I can do either.